src/docs/development/packages-and-plugins/plugin-api-migration.md

```diff
@@ -134,7 +130,7 @@ public class EmbeddingV1Activity extends FlutterActivity {
 8. To have the plugin support Flutter on branches master and stable,
```

Comment:
we can remove this whole step by the time stable is out.

Reply:
lets revisit once the dust has settled